apple-nav/src/components/Subnav.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import "./Nav.css";
+import { Route, Link, NavLink } from "react-router-dom";
+
+function Subnav(props) {
+  const item = props.navList.find(
+    item => `${item.path}` === props.match.params.path
+  );
+  if (!item) return <h3>Loading data...</h3>;
+  return (
+    <div>
+      <nav className={`sub-nav${item.name === "TV" ? " tv" : ""}`}>
```

Comment:
great job conditionally rendering the className. you're really getting this down!